package.json

```diff
@@ -54,6 +54,7 @@
 		"typescript:build": "node npm-scripts.mjs typescript:build",
 		"typescript:watch": "node npm-scripts.mjs typescript:watch",
 		"worker:build": "node npm-scripts.mjs worker:build",
+		"worker:build-lib": "node npm-scripts.mjs worker:build-lib",
```

Comment:
As expected, this is not needed at all, please remove.

Reply:
I've been using it while development, left over. Remember this is still a DRAFT.